arm-compute/2016-09-30/swagger/containerService.json

```diff
@@ -0,0 +1,595 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerServiceClient",
+    "description": "The Container Service Client.",
+    "version": "2016-09-30"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
```

Comment:
does the service actually support "text/json"? I am asking this because storage management spec has the same stuff and it sends a 409 when content-type is set to text/json. Hence want to be sure if this is actually supported by the service.